pkg/tfbridge/schema_test.go

```diff
@@ -990,6 +990,28 @@ func TestInvalidAsset(t *testing.T) {
 	}, outputs)
 }
 
+func TestOverridingTFSchema(t *testing.T) {
+	result := MakeTerraformOutputs(
+		map[string]interface{}{
+			"my_string_property_value": MyString("true"),
+		},
+		map[string]*schema.Schema{
+			"my_string_property_value": {Type: schema.TypeString},
+		},
+		map[string]*SchemaInfo{
+			"my_string_property_value": {
+				Type: "boolean",
+			},
+		},
+		nil,   /* assets */
+		false, /*useRawNames*/
+		true,
+	)
+	assert.Equal(t, resource.NewPropertyMapFromMap(map[string]interface{}{
+		"myStringPropertyValue": true,
+	}), result)
+}
+
```

Comment:
What happens with the empty string?  I believe the motivation for this change upstream is to support “undefined” as well as true and false as values. Is that true?  And if so - how does it manifest in practice in providers (and as a result - what do we need to support here)?

Reply:
if we pass an override as an empty string then we just return the string - the same behaviour as the default case of the TF provider

Reply:
Sorry - my question was about `""` coming from the provider as the value of the property.  So what happens if `""` is the value in the first argument to `TestOverridingTFSchema`. And ultimately will that show up sensibly in Pulumi SDKs (as an undefined value for an optional output I guess)?

Really I'm just interested in making sure this really works for the actual scenarios where we need to map into and out of Terraform resources that use this (across the range of values those provider can in practice accept or return).